tqdm/cli.py

```diff
@@ -1,6 +1,8 @@
 """
 Module version for monitoring CLI pipes (`... | python -m tqdm | ...`).
 """
+from __future__ import absolute_import
```

Comment:
this is only required for Python 2 support, if we keep the `logging` submodule name